.github/workflows/a11y-add-alt-bot.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+  accessibility_alt_text_bot:
+    if: ${{ github.event.issue || github.event.pull_request || github.event.discussion }}
+    name: Check alt text is set on issue or pull requests
```

Comment:
[Cleanup] This repo's workflows generally omit `name` when it doesn't add much. IMO this one can be omitted.

```suggestion
```